src/conda_patches/0002-Manipulate-PATH-directly-instead-of-_call_ing-conda.patch

```diff
@@ -1,26 +1,27 @@
-From a3edb498065b15298ebb298492c83d1124b36138 Mon Sep 17 00:00:00 2001
 From: Nehal J Wani <nehaljw.kkd1@gmail.com>
 Date: Tue, 13 Aug 2019 07:46:10 -0400
 Subject: [PATCH 2/2] Manipulate PATH directly instead of _call_ing conda.bat
-
----
- conda/utils.py | 3 ++-
- 1 file changed, 2 insertions(+), 1 deletion(-)
+Updated: 2022-12-14 by @jaimergp, conda 22.11.1 compatibility
 
 diff --git a/conda/utils.py b/conda/utils.py
-index 4b689f888..bd940491c 100644
+index b39912282..c3a2a8e89 100644
 --- a/conda/utils.py
 +++ b/conda/utils.py
-@@ -362,7 +362,8 @@ def wrap_subprocess_call(on_win, root_prefix, prefix, dev_mode, debug_wrapper_sc
+@@ -395,7 +395,16 @@ def wrap_subprocess_call(
              # after all!
              # fh.write("@FOR /F \"tokens=100\" %%F IN ('chcp') DO @SET CONDA_OLD_CHCP=%%F\n")
              # fh.write('@chcp 65001>NUL\n')
--            fh.write('{0}CALL \"{1}\" activate \"{2}\"\n'.format(silencer, conda_bat, prefix))
-+            # fh.write('{0}CALL \"{1}\" activate \"{2}\"\n'.format(silencer, conda_bat, prefix))
-+            fh.write('SET PATH={p};{p}\\Library\\mingw-w64\\bin;{p}\Library\\usr\\bin;{p}\\Library\\bin;{p}\\Scripts;{p}\\bin;%PATH%'.format(p=prefix))
-             fh.write("{}IF %ERRORLEVEL% NEQ 0 EXIT /b %ERRORLEVEL%\n".format(silencer))
+-            fh.write(f'{silencer}CALL "{conda_bat}" activate "{prefix}"\n')
++            # fh.write(f'{silencer}CALL "{conda_bat}" activate "{prefix}"\n')
++            fh.write(
++                f'SET "PATH={prefix};'
++                f'{prefix}\\Library\\mingw-w64\\bin;'
++                f'{prefix}\\Library\\usr\\bin;'
++                f'{prefix}\\Library\\bin;'
++                f'{prefix}\\Scripts;'
++                f'{prefix}\\bin;'
++                '%PATH%"'
++            )
+             fh.write(f"{silencer}IF %ERRORLEVEL% NEQ 0 EXIT /b %ERRORLEVEL%\n")
```

Comment:
I'd really love to understand this patch!

Reply:
Same :D 

Earliest [history bit](https://github.com/AnacondaRecipes/conda-standalone-feedstock/commit/ae5b5ef6411531c71eac776881951d1451b7f082) I could find already included this file without explanation, so I am guessing it comes from a different repot (the patch date predates the repo creation date).

@nehaljwani - do you recall any details?

Reply:
IIRC, there is no `conda.bat` file to call activate on, when it is invoked in the frozen state.

Reply:
There's no corresponding unix patch though.